### PR TITLE
update vendor

### PR DESCRIPTION
**Pull request by hicqu**
It's for document-store.

### CODE REVIEW

**Comment by hicqu** [comment]
PTAL, thanks.

**Comment by hanfei1991** [comment]
LGTM

File: glide.lock [hunk 1]
```diff
@@ -62,7 +62,7 @@ imports:
 - name: github.com/golang/snappy
   version: 723cc1e459b8eea2dea4583200fd60757d40097a
 - name: github.com/gorilla/context
-  version: 1ea25387ff6f684839d82767c1733ff4d4d15d0a
+  version: 08b5f424b9271eedf6f9f0ce86cb9396ed337a42
```
Comment (tiancaiamao) [inline glide.lock]: I think you should put it in glide.yaml

File: glide.lock [hunk 2]
```diff
@@ -180,7 +180,7 @@ imports:
   - encoding/unicode
   - transform
 - name: golang.org/x/time
-  version: c06e80d9300e4443158a03817b8a8cb37d230320
+  version: a4bde12657593d5e90d0533a3e4fd95e635124cb
```
Comment (tiancaiamao) [inline glide.lock]: ditto

**Comment by hicqu** [comment]
now only tipb is updated. other packages keep original version. PTAL again, thanks!

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**